consensus/replay.go

```diff
@@ -46,6 +47,18 @@ func (cs *ConsensusState) readReplayMessage(msgBytes []byte, newStepCh chan inte
 		fmt.Println("MsgBytes:", msgBytes, string(msgBytes))
 		return fmt.Errorf("Error reading json data: %v", err)
 	}
+	// check checksum
+	innerMsgBytes := wire.JSONBytes(msg.Msg)
+	crc32c := crc32.MakeTable(crc32.Castagnoli)
```

Comment:
Minor nit: if this function is in a hot path, we can declare as a global `var crc32c = crc32.MakeTable(crc32.Castagnoli)`

Reply:
nice idea